packages/sync/legacy/class.jetpack-sync-users.php

```diff
@@ -9,7 +11,8 @@ class Jetpack_Sync_Users {
 	static $user_roles = array();
 
 	static function init() {
-		if ( Jetpack::is_active() ) {
+		$connection_manager = new Connection_Manager();
```

Comment:
Same here, can we please pass the same object around?

Reply:
Sure 👍 See https://github.com/Automattic/jetpack/pull/12665#discussion_r292876721